### DISCUSSION TITLE
How to provide metrics data via opentelemetry to SkyWalking OAP

### DISCUSSION THREAD

**Discussion by leopardrota** [apache/skywalking#10793] [Q&A] [answered]
I want to report metrics data via opentelemetry to SkyWalking OAP in a Java project. Can someone provide a demo or related guidance related or something like that? Thanks!

**Comment by wu-sheng** [comment]
What is a strange asking. Why metrics are required via OTLP?

**Reply by leopardrota** [reply]
My poor English, my words don't convey my meaning. I try to use other words: metrics in opentelemetry format.

**Reply by wu-sheng** [reply, marked as answer]
How to collect metrics is your responsibility as you chose OTLP. OAP has provided OTLP metrics receiver for this format.
Ref, https://skywalking.apache.org/docs/main/next/en/setup/backend/opentelemetry-receiver/
